src/htmx.js

```diff
@@ -864,13 +864,12 @@ return (function () {
             if (oobSelects) {
                 var oobSelectValues = oobSelects.split(",");
                 for (var i = 0; i < oobSelectValues.length; i++) {
-                    var oobSelectValue = oobSelectValues[i].split(":", 2);
-                    var id = oobSelectValue[0].trim();
-                    if (id.indexOf("#") === 0) {
-                        id = id.substring(1);
-                    }
-                    var oobValue = oobSelectValue[1] || "true";
-                    var oobElement = fragment.querySelector("#" + id);
+                    const oobSelectValue = oobSelectValues[i]
+                    // Support colon in css selectors
+                    const colon = oobSelectValue.lastIndexOf(':')
+                    const split_at = colon == -1 ? oobSelectValue.length : colon
+                    const oobValue = oobSelectValue.substring(split_at + 1) || 'true'
+                    const oobElement = fragment.querySelector(oobSelectValue.substring(0, split_at))
```

Comment:
unfortunately in v1 we can't use const, since it needs to be IE compatible.  Can you revert to var?